### PR TITLE
Migrating scalability kubemark presubmits to clusterloader

### PR DESCRIPTION
**Pull request by krzysied**
Migrating scalability kubemark presubmits to clusterloader.

### CODE REVIEW

File: config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml [hunk 1]
```diff
@@ -219,7 +258,15 @@ presubmits:
         - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-kubemark-e2e-gce-big
         - --tear-down-previous
         - --test=false
-        - --test_args=--ginkgo.focus=\[Feature:Performance\] --gather-resource-usage=true --gather-metrics-at-teardown=true
+        - --test-cmd=$GOPATH/src/k8s.io/perf-tests/run-e2e.sh
+        - --test-cmd-args=cluster-loader2
+        - --test-cmd-args=--nodes=2000
```
Comment (wojtek-t) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml]: 500

Anyway - I think we have too many jobs here. What's the point of having one jobs per release, and one for all 4 releases? @krzyzacy @BenTheElder ?

File: config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml [hunk 2]
```diff
@@ -16,6 +16,7 @@ presubmits:
       - args:
         - --root=/go/src
         - --repo=k8s.io/kubernetes=$(PULL_REFS)
+        - --repo=k8s.io/perf-tests=master
```
Comment (wojtek-t) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml]: Can you please split just non-kubemark presubmits to a separate PR? I would like to understand the reasoning behing 5 jobs for kubemarks first.

Comment (krzysied) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml]: #11171 created.

Comment (krzysied) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml]: I removed gce to #11171. Only kubemark presubmits left.

File: config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml (hunk 1 above)
Comment (wojtek-t) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml]: I already figured this out - those differ by image which depends on the branch.

**Comment by wojtek-t** [comment]
/lgtm
/approve

/hold
Let's not merge on Friday afternoon - we will merge that on Monday morning.

**Comment by wojtek-t** [comment]
/assign @krzyzacy 
/assign @BenTheElder 

for approval

**Comment by wojtek-t** [comment]
/hold cancel

Let's try this now.

**Pull request merged**